t/plugin/proxy-rewrite3.t

```diff
@@ -200,3 +200,103 @@ passed
 GET /hello
 --- error_log
 plugin_proxy_rewrite get method: POST
+
+
+
+=== TEST 8: set route(unsafe uri normalized at request)
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "methods": ["GET"],
+                        "plugins": {
+                            "proxy-rewrite": {
+                                "use_real_request_uri_unsafe": false
```

Comment:
It looks like we don't need this test case("use_real_request_uri_unsafe": false) because it defaults to false and there are existing test cases that override the default behavior.